testui/elements/testui_element.py

```diff
@@ -205,41 +210,40 @@ def visible_for(self, seconds=1):
                            f'for {seconds}s')
         return self
 
-    def wait_until_visible(self, seconds=10.0, log=True):
+    def wait_until_visible(self, seconds=10.0, log=True) -> "Elements":
         start = time.time()
-        err = None
-        err_text = 'not'
+
+        is_not = self.__is_not
+
+        # Executes the loop at least once in case `seconds` <= 0
+        entered_loop = False
+        while time.time() < start + seconds or not entered_loop:
+            if self.is_visible(log=False):
+                if log:
+                    self.__put_log(
+                        f'{self.device_name}: Element "{self.locator_type}: {self.locator}" '
+                        f"was visible in {time.time() - start}s"
+                    )
+
+                return self
+
+            self.__is_not = is_not
+            entered_loop = True
+            
+            time.sleep(0.2)
+
+        err_text = "was not"
         if self.__is_not:
-            err_text = 'is'
-        i = 0
-        while time.time() < start + seconds or i < 1:
-            try:
-                self.get_element()
-                if not self.__is_not:
-                    if log:
-                        self.__put_log(f'{self.device_name}: element "{self.locator_type}: {self.locator}" '
-                                       f'found after {time.time() - start}s')
-                    self.__is_not = False
-                    return self
-            except Exception as error:
-                if self.__is_not:
-                    if log:
-                        self.__put_log(f'{self.device_name}: element "{self.locator_type}: {self.locator}" not visible '
-                                       f'after {time.time() - start}s')
-                    self.__is_not = False
-                    return self
-                err = error
-                time.sleep(0.2)
-            i += 1
-        if err is None:
-            err = ''
-        if log:
-            return self.__show_error(
-                f'{logger.bcolors.FAIL}{err} {self.device_name} Element {err_text} found with the following locator: '
-                f'"{self.locator_type}: {self.locator}" after {time.time() - start}s {logger.bcolors.ENDC}'
```

Comment:
Because `self.__show_error()` wraps `testui_error()` which in the end calls `logger.log_error()` that has these colors like so
```py
def log_error(message, use_date=False):
    now = datetime.now()
    current_time = now.strftime("%Y-%m-%d %H:%M:%S")
    date = ''
    if use_date:
        date = f'[{current_time}] '
    print(f"{bcolors.FAIL}{date}{message}{bcolors.ENDC}")
    file = open(__file_log(), 'a+')

    file.write(f'{date}{message}\n')
    file.close()
```